### ISSUE TITLE
Waveform

### ISSUE THREAD

**Issue by ericraio** [jhabdas/lumpen-radio#5]
In my radio app I am using StreamingKit for iOS and MediaPlayer for Android, I will be adding a waveform feature to display neat looking audio waves onto the screen.

I just started conversations with the owner of https://github.com/vespakoen/react-native-waveform

Any interest in contributing to this lib and adding it to lumped radio?

**Comment by jhabdas** [comment]
@ericraio what a cool project! I'd love to help but I'm absorbed in other things right now. if and when I get a chance to enhance Lumpen Radio I'd like to add AirPlay support for AppleTV 4 and maybe extend the app for use on tvOS and possibly the watch. For visualizations what I'd like to do is create a full-screen visualizer which responds to music in real-time using beat matching and whatnot. I saw some cool JS libs out there to do such a thing, but can't seem to dig up the best one I saw right now. thanks for the offer and good luck!
